src/event-handlers.js

```diff
@@ -399,6 +399,17 @@ export default {
     this.setState({ useAlternateEndDayButtonPosition })
   },
 
+  handleHideHomeScreenChange(
+      _e,
+      hideHomeScreen
+  ) {
+    if (this.state.stageFocus === 'HOME') {
```

Comment:
instead of the string 'HOME' here, i think you'll want to use `stageFocusType.HOME` from enums instead. if you look into the `Farmhand.js` file you can see it used there, but just let me know if you don't find it

Reply:
Ack yeah, I should have caught that sorry. 🤦‍♀️ Fixed!